public/app/features/library-panels/components/LibraryPanelCard/LibraryPanelCard.tsx

```diff
@@ -48,3 +52,50 @@ export const LibraryPanelCard: React.FC<LibraryPanelCardProps & { children?: JSX
     </>
   );
 };
+
+interface FolderLinkProps {
+  libraryPanel: LibraryPanelDTO;
+}
+
+function FolderLink({ libraryPanel }: FolderLinkProps): JSX.Element {
+  const styles = useStyles2(getStyles);
+  const onNavigateToFolder = (e: MouseEvent<HTMLButtonElement | HTMLAnchorElement>) => {
+    e.preventDefault();
+    locationService.push(urlUtil.renderUrl(`/dashboards/f/${libraryPanel.meta.folderUid}`, {}));
+  };
+
+  if (!libraryPanel.meta.folderUid) {
+    return (
+      <span className={styles.metaContainer}>
+        <Icon name={'folder'} size="sm" />
+        {libraryPanel.meta.folderName}
+      </span>
+    );
+  }
+
+  return (
+    <a href="#" onClick={onNavigateToFolder}>
```

Comment:
Think you can use `Link` component from react-router-dom to avoid the onNavigateToFolder callback

Reply:
Aha nice, didn't know about that. Thank you!

Reply:
why is not a normal url used here and instead onClick used?  the url should on the anchor should take precedence over the card click I think

Reply:
You're probably right Torkel, I'll try it out.

Reply:
Fixed